web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -292,38 +427,72 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
             label="Name"
             placeholder="Period Name"
             required
-            value={periodToAdd.name ? periodToAdd.name : ""}
-            onChange={(e) =>
-              setPeriodToAdd({ ...periodToAdd, name: e.target.value })
-            }
-          />
-          <TextField
```

Comment:
This TextField was replaced by a Select component that is wrapped in a FormControl.